api/client/batch_client.py

```diff
@@ -37,6 +39,15 @@ def get_data(self, url, headers, params=None):
         base_log_record = dict(route=url, params=params)
         retry_count = 0
         self._logger.debug(url)
+
+        # append version info
```

Comment:
minor point; to avoid duplicating these few lies of code you can just have a function 
lib.get_version_info() return a dicts, and then do 

headers.update(lib.get_version_info())

Reply:
done